components/src/core/components/Icon/icons.ts

```diff
@@ -377,6 +377,12 @@ export const oxdCalendar: icon = {
 };
 
+export const oxdNewFolder: icon = {
+  name: 'oxd-filter',
```

Comment:
name should not be oxd-filter right?

Reply:
yes it should not be. I updated it.. Thank you